robot.py

```diff
@@ -21,7 +20,6 @@ class MyRobot(magicbot.MagicRobot):
     vision: Vision
     arm: Arm
     intake: Intake
-    gripper: Gripper
```

Comment:
???

Reply:
Yeah that was a big mistake. I think I accidentally deleted the import line and this.
Will address this in a moment.